src/logic/embedding/basic.lean

```diff
@@ -65,6 +65,7 @@ instance equiv.coe_embedding : has_coe (α ≃ β) (α ↪ β) := ⟨equiv.to_em
 instance equiv.perm.coe_embedding : has_coe (equiv.perm α) (α ↪ α) := equiv.coe_embedding
 
 @[simp] lemma equiv.coe_eq_to_embedding  : ↑f = f.to_embedding := rfl
+@[simp] lemma equiv.coe_to_embedding : ⇑f.to_embedding = f := rfl
```

Comment:
I'd argue this belongs on line 61; it's not  about the coercion instance that the lemma above is.

Reply:
(also, I'm surprised the `simps` lemma doesn't cause the linter to reject this)

Reply:
Weirdly enough, this new simp lemma is needed on branch `additive_energy`.